model/access_test.go

```diff
@@ -20,9 +20,7 @@ func TestAccessJson(t *testing.T) {
 	json := a1.ToJson()
 	ra1 := AccessDataFromJson(strings.NewReader(json))
 
-	if a1.Token != ra1.Token {
-		t.Fatal("tokens didn't match")
-	}
+	require.Equal(t, a1.Token, ra1.Token, "tokens didn't match")
```

Comment:
When the text message is obvious enough from the assertion is not needed. In this case we are comparing the tokens for equality, and is not matching, so the default error message is going to be enough, you don't need an specific one.

The same can be applied to the rest of your changes.